spring-kafka/src/main/java/org/springframework/kafka/config/AbstractKafkaListenerContainerFactory.java

```diff
@@ -60,6 +65,14 @@
 
 	private DeDuplicationStrategy<K, V> deDuplicationStrategy;
 
+	private Boolean pauseEnabled;
+
+	private Long pauseAfter;
```

Comment:
Do we even use this value here? We have another one, with a default, below: https://github.com/spring-projects/spring-kafka/pull/91/files/fa9fd7286cd54c888774b65fcbef5eab5f92e493#diff-8ebdce1dc0deea04032bdfadbcd7bb4aR130

Wondering if we should externalize all configuration properties and pass the configured instance from the `Concurrent` class to the simple one. Every time we add property we need to copy it.